lib/diagnostic.js

```diff
@@ -39,14 +43,20 @@ const owner = null
 // Methods: vocalize
 
 // Replace null with your answer
-const pet = null
+const pet = {name: "Rover", species: "dog", breed: "Golden Retriever", vocalize: "woof"};
+sound = function(){return pet.vocalize};
 
 // 4. Create a constructor function for pet objects, as described above. The
 // constructor function should be responsible for setting the values of `name`,
 // `species`, `breed`, and `vocalization`.
 
 const Pet = function () {
```

Comment:
You want this function to accept these as parameters, so something like `const Pet = function(name, species, breed, vocalize){...`